frontend/src/components/TextStyles.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components'
+
+export const H1 = styled.h1`
+  text-align: center;
+  color: white;
+`
+export const H2 = styled.h2`
+  text-align: center;
+  color: white;
+  font-size: 24px;
+`
+
+export const P = styled.p`
+  font-size: 16px;
+  color: white;
+  `
```

Comment:
One of the best things with styled components is that you can name them more descriptively than html tags in my opinion. These could be called Headline and Text or BodyText or something